projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -651,6 +654,28 @@ public function validate_themes_param( $themes ) {
 		return true;
 	}
 
+	/**
+	 * Checks that the "paths" parameter is a valid array of paths.
+	 *
+	 * @param array $paths List of paths to check.
+	 * @return bool|WP_Error
+	 */
+	public function validate_paths_param( $paths ) {
```

Comment:
I'm wondering if we should do more validation here, for example, we probably want to allow users to create both absolute URLs and relative URLs, if they use absolute URLs, we will need to make sure the domain matches their site. I'm also not sure if we should do a header check at this stage, otherwise user might input something that's 404 in the first place, what do you think? One caveat of doing the header check here is that it might slow down the original action.  Or we should just let it pass through and wait until later?